docs/CLI.md

```diff
@@ -41,6 +41,13 @@ Tasks and variants can also be specified using the regex variants(-rv) and regex
 evergreen patch --regex_variants "enterprise.*" --rt "test-.*"
 ```
 
+Tasks and variants must _both_ be specified:
+```bash
+evergreen patch --regex_variants "enterprise.*" // not valid, will not select any tasks
+evergreen patch -t unittest // not valid, will not select any tasks
+evergreen patch --regex_variants "enterprise.*" -t unittest // valid
```

Comment:
nit, can we use `regex_tasks` instead of `-t` to match it to `regex_variants`?

Reply:
I purposely didn't match them to show that this works even if you mix regex tasks and variants. I'll add another example with both regex tasks and variants though.